application/src/main/java/org/thingsboard/server/service/edge/rpc/sync/DefaultEdgeRequestsService.java

```diff
@@ -163,6 +164,9 @@ public void onSuccess(@Nullable List<AttributeKvEntry> ssAttributes) {
                     Map<String, Object> entityData = new HashMap<>();
                     ObjectNode attributes = mapper.createObjectNode();
                     for (AttributeKvEntry attr : ssAttributes) {
+                        if (DefaultDeviceStateService.PERSISTENT_ATTRIBUTES.contains(attr.getKey())) {
```

Comment:
maybe not contains?

Reply:
no, persistent attributes should not be propagated to edge.
active and lastActivityTime are updated by edge respectively on any timeseries data from the cloud or locally.